viya_ark_library/k8s/test_impl/response_data/resources_secrets.json

```diff
@@ -0,0 +1,102 @@
+[
+        {
+            "apiVersion": "v1",
+            "data": {
+                "permissions": "Kg==",
+                "rolename": "cGdvYWRtaW4="
```

Comment:
These look protected by k8s as well, would these be set to something like `"<permissions>"` and `"<rolename>"` just to negate the chance that any sensitive data is included.